Save/rental_storage.rb

```diff
@@ -0,0 +1,47 @@
+require 'json'
+require_relative './storage'
+require './rental'
+require './Save/book_storage'
+require './Save/people_storage'
+
+class RentalStorage < Storage
+  @rentals = []
+
+  class << self
+    attr_accessor :rentals
+  end
+
+  def self.fetch
+    rentals = if File.exist?('./data/rentals.json')
+                JSON.parse(File.read('./data/rentals.json'))
+              else
+                []
+              end
+    deserialize(rentals)
+  end
+
+  def self.save(rentals)
+    rentals.each do |rental|
+      RentalStorage.rentals.push(serialize(rental))
+    end
+
+    File.write('./data/rentals.json', JSON.pretty_generate(RentalStorage.rentals))
+  end
+
+  def self.serialize(rental)
+    {
+      date: rental.date,
+      book: BookStorage.serialize(rental.book),
+      person: PeopleStorage.serialize(rental.person)
+    }
+  end
+
+  def self.deserialize(item)
+    temp = []
+    item.each do |rental|
+      new_rental = Rental.new(rental['date'], rental['book'], rental['person'])
+      temp.push(new_rental)
+    end
```

Comment:
- [ ] When I try to renter the application after saving the data in JSON files, I get the following errors

![image](https://user-images.githubusercontent.com/77106962/232890566-005d7d28-32bc-4b0e-941a-4a1956059a9d.png)

#### Upon entry, when you are creating a new instance of the `Rental` class, they have no `rentals` array for the `Person` object or `Book` object retrieved from the JSON files and it is specified in the `initialize` method of the `Rental` class, that there should be a `rentals` array data. It would be nice to fix this. 👍 😎